etc/config-template.toml

```diff
@@ -64,6 +64,9 @@
 ## If not set, `addr` will be used.
 # advertise-addr = ""
 
+## HTTP listening address.
+# http-addr = "127.0.0.1:0". If it is not specified, the system will use a random port.
```

Comment:
Maybe we can just add another config called `enable-http-server`. BTW I think the config should be grouped outside the `[server]` config cluster, because tikv::server actually does not utilize it, like:

```toml
[http_server]
# enabled = false
# bind = "127.0.0.1:10081"
```

Reply:
I don't think we need another config. If we don't configure the HTTP address, we don't need to start HTTP in TiKV.